devel/go-tools/Portfile

```diff
@@ -3,8 +3,8 @@
 PortSystem          1.0
 PortGroup           github 1.0
 
```

Comment:
This port could probably benefit from using the `golang` PortGroup.

It knows how to compile Go software in a platform-portable way across older versions of macOS, among other benefits.

By default we want to use it for any port for software written in Go.

We'd start off here with:
```
PortGroup           golang 1.0
```